Services/Services.Domain/Entities/BaseEntity.cs

```diff
@@ -10,6 +10,6 @@ namespace Services.Domain.Entities
     public class BaseEntity
     {
         [Key]
-        public Guid Id { get; set; }
+        public Guid Id { get; set; } = Guid.NewGuid();
```

Comment:
I don't think you need this, this should be created automatically in the db

Reply:
Fixed